sig-contributor-experience/charter.md

```diff
@@ -1,58 +1,141 @@
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+The [Contributor Experience Special Interest Group] (SIG) is responsible for improving the experience of those who upstream contribute to the Kubernetes project. We do this by creating, and maintaining programs and processes that promote community health and reduce project friction, while retiring those programs and processes that don't. Being conscientious of our contributor base is critical to scaling the project, growing the ecosystem, and helping the project succeed.
+
+We do this by listening - whether it’s through our roadshows to SIG meetings, surveys, data, or [GitHub issues], we take in the feedback and turn it into our [project list]. We build a welcoming and inclusive community of contributors by giving them places to be heard and productive.
+
+### In scope
+
+#### Code, Binaries and Services
+
+- Establish policies, standards, and procedures:
+  - routine GitHub management tasks, including but not limited to: org membership, org permissions, repo creation/administration.
+  - "Org Owner" GitHub permissions, and grant/limit these privileges accordingly
+  - bot accounts, service accounts, webhooks, and third-party integrations for all communication platforms that we support but most importantly, GitHub.
+- Establishing a "GitHub Administration team" that will oversee the execution of GitHub management tasks: inviting new members to the org, creating repos, executing moderation decisions, auditing permissions.
+- Work with other SIGs and interested parties in the project to execute GitHub tasks where required
+-Own and execute events that are targeted to the Kubernetes contributor community, including:
+  - The weekly [Kubernetes Community meeting]
+  - [Contributor Summit(s)]
+  - SIG Contributor Experience face to face meetings
+  - [Steering Committee elections] (though we do not own policy creation, see 'out of scope' below)
+  - Retrospective moderation for other SIGs upon request
+  - Other events, like other SIG face to face events, upon request and consideration
+- Strategize, build, and execute on scalable [mentoring programs] for all contributor levels. These may include:
+  - [Google Summer of Code]
+  - [Outreachy]
+  - [Meet Our Contributors]
+  - [Group Mentoring - WIP]
+  - [The 1:1 Hour - WIP]
+  - Speed Mentoring at KubeCon
+- Create best practices, trainings, policies, and [moderation] of Kubernetes communication platforms, the transparency vehicles of information throughout the project. This includes user guidelines when the contributor is the user of the platform. Moderation is an always on, never off service that we supply with trusted contributors spanning time zones. This includes immediate action when dealing with code of conduct issues in a public space that are defined below:
```

Comment:
This is pretty long for a point form list, and mixes both owning the platforms, and doing moderation on them. Perhaps this can be shortened or split up?

Also, we (via the github admin team) would handle moderation on GitHub too, where that's needed.

Reply:
much clearer! 👍